integration/js/app/package-lock.json

```diff
@@ -55,6 +55,7 @@
         "fs-extra": "^8.1.0",
         "git-rev-sync": "^3.0.2",
         "longjohn": "^0.2.12",
+        "markdown-it": "^13.0.1",
```

Comment:
This PR seems to add a new dependency `markdown-it` to the integration test. Can you confirm?

Reply:
I think this came when I moved the `markedown-it` from `dependencies` to `devDependencies` in last PR. Checked and removing this as we do not need in JS SDK but use it in `demos/browser` only. Thanks for the catch!

Reply:
Updated.